src/ucp/core/ucp_request.c

```diff
@@ -746,6 +746,11 @@ ucs_status_t ucp_request_progress_wrapper(uct_pending_req_t *self)
                   req->send.state.dt_iter.length);
 
     ucs_log_indent(1);
+
+    if (ucs_unlikely(ucp_worker_should_track_usage(req->send.ep->worker))) {
+        ucp_worker_track_ep_usage(req);
+    }
```

Comment:
maybe unite it to one inline function? the inline part will check TIMER_SKIP and USAGE_TRACEKED flag. Probably also need to check if usage tracker is enabled.
The non-inline part will do all the rest.

Reply:
what is the logic behind this division? why not make all the function inline?